src/status_im/ui/components/desktop/tabs.cljs

```diff
@@ -26,7 +27,7 @@
     [react/view {:style tabs.styles/tab-container}
      (let [icon (if active? icon-active icon-inactive)]
        [react/view
-        [icons/icon icon {:color (:color (tabs.styles/tab-icon active?))}]])
+        [icons/icon icon {:style {:tint-color (if active? "#4360df" "#6e777e")}}]])
```

Comment:
@siphiuel, probably here we also should use colors namespace.

Reply:
@vkjr I thought you wouldn't notice :) Fixed.